test/integration/dialects/mysql/query-generator.test.js

```diff
@@ -369,6 +369,11 @@ if (Support.dialectIsMySQL()) {
           arguments: ['myTable', {where: {field: {ne: null}}}],
           expectation: 'SELECT * FROM `myTable` WHERE `myTable`.`field` IS NOT NULL;',
           context: QueryGenerator
+        }, {
```

Comment:
in sql/where.test.js or sql/select.test.js - I know the bug is mysql specific, but we might as well add unit testing for all dialects